cli/kata-env.go

```diff
@@ -11,19 +11,20 @@ import (
 	"os"
 	"strings"
 
+	runtim "runtime"
```

Comment:
The rename is weird....

Reply:
I don't like that either. But to be fair, @devimc didn't do this - the reformatter simply moved it if you look below.